lib/fluent/plugin/out_remote_syslog.rb

```diff
@@ -43,8 +43,8 @@ def emit(tag, es, chain)
         tag = rewrite_tag!(tag.dup)
         @loggers[tag] ||= RemoteSyslogLogger::UdpSender.new(@host,
           @port,
-          facility: @facility,
-          severity: @severity,
+          facility: record['facility'] || @facility,
```

Comment:
> Align the parameters of a method call if they span more than one line.

sory..
I can not understand what you mean.
※ I am not good at ruby and English.